### DISCUSSION TITLE
Templates for default values

### DISCUSSION THREAD

**Discussion by fredyk** [fredyk/westack-go#167] [Ideas]
@gmontielg I think we could process the following default value templates. Each one would be parsed depending on its associated type:

```json
  "type": "date",
  "default": "now"
```

```json
  "type": "date",
  "default": "-<x>s"
```
&lt;x&gt;: some integer, s: seconds

```json
  "type": "date",
  "default": "+<x>s"
```

In addition, I'm wondering whether these templates should be prefixed with a `$` or something similar. Or we could reserve the `$` for environment/in-code variables.
What do you think about?

#164

**Comment by gmontielg** [comment]
Looks great! let's use those templates.
